tests/test_units/test_contents.py

```diff
@@ -0,0 +1,261 @@
+"""Explicitly test all allowed contents."""
+
+from fmu.dataio.dataio import ExportData
+
+# generic testing of functionality related to content is done elsewhere,
+# mainly in test_dataio.py.
+
+
+def test_content_facies_thickness(regsurf, globalconfig2):
+    """Test export of the facies_thickness content."""
+    meta = ExportData(
+        config=globalconfig2,
+        name="MyName",
+        content="facies_thickness",
+    ).generate_metadata(regsurf)
+
```

Comment:
Since the structure/pattern of all tests are rather similar, have you considered one test in staed with decorator `@pytest.mark.parametrize()`?

Reply:
Yes, I considered it but skipped it as I expect there to be more tests for each content as we go along. So for readability, the idea was to bundle tests related to the same content together, rather than bundling tests with similar patterns together. But that's up for debate.